src/Umbraco.Web.UI/Umbraco.Web.UI.csproj

```diff
@@ -67,6 +67,9 @@
   <!-- We don't want to include the generated files, they will throw a lot of errors -->
 
   <ItemGroup>
+    <PackageReference Include="GoogleAuthenticator" Version="2.4.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Authentication.Google" Version="5.0.13" />
+    <PackageReference Include="Microsoft.AspNetCore.Authentication.Twitter" Version="5.0.13" />
```

Comment:
```suggestion
```

Don't think we want to include these by default? I assume it's an accidental commit :D